.eslintrc.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
 		ecmaVersion: 2018,
 	},
 	rules: {
-		'no-warning-comments': 'warn',
+		'no-warning-comments': [0, { 'terms': ['todo'], 'location': 'start' }],
```

Comment:
this way todo doesn't get triggered, since `0` is disabled this means todo or whatever words we add here wont trigger anything.

Reply:
this setting is not enabled by default so `[0, { 'terms': ['todo'], 'location': 'start' }]` is the same thing as removing the rule.

I wish there was a way to have different levels for the same rule with different options but it seems eslint doesn't want to work on that https://github.com/eslint/eslint/issues/11089

Reply:
I dont see the value in these warnings, but perhaps you have some reason to enable this I dont see at this time., I can also update and do this.
```suggestion
```

Reply:
I'm ok with removing the rule.